### PR TITLE
update keepers with new explanations

### PR DESCRIPTION
**Pull request by StephenFluin**
This PR intentionally does not improve everything, this is a small first step. Additional updates will come as separate PR so they can be evaluated independently.

### CODE REVIEW

File: docs/Keepers/introduction.md [hunk 1]
```diff
@@ -2,28 +2,54 @@
...
 {% include keepers-beta %}
 
 A major limitation of smart contracts is that they can't trigger or initiate their own functions at arbitrary times or under arbitrary conditions. State change will only occur when a transaction is initiated by another account (such as user, oracle, or contract).
 
-Chainlink Keepers allow you to register Upkeep for your contract. When the conditions you specify are met, the Chainlink Keeper Network will execute a method on your contract. [Learn how the network works.](../overview)
+To solve this problem, blockchain projects can:
+
+1. Create highly reliable infrastructure trigger smart contract functions - centralized and expensive to build/maintain,
+1. Outsource it to a 3rd party - possibly centralized or having a single point of failure in an external party, or with complex incentive alignment
+1. Use the Chainlink Keeper Network
```
Comment (AnushV) [inline docs/Keepers/introduction.md:23]: The changes here miss the target a bit. I've been talking to a number of projects recently and I think the distinction between 3rd party outsourcing and open systems are two independent concepts that are well recognized. The open market solutions have reliability concerns hence why Chainlink Keepers is attractive.

2. Outsource it to a 3rd party - often centralized and a single point of failure in an external party
3. Use an open market solution - decentralized, but comes along with complex incentive alignment, potential for competitive bots that increase the cost of execution, and ensuring reliability can be difficult
4. Use Chainlink Keeper Network

**Review by AnushV — CHANGES_REQUESTED** [review]
Made one comment that needs to be addressed. Once that is done I think this change is read to go.

File: docs/Keepers/introduction.md (hunk 1 above)
Comment (StephenFluin) [reply docs/Keepers/introduction.md:24]: updated with
```
1. Create highly reliable infrastructure trigger smart contract functions - centralized and expensive to build/maintain,
1. Outsource it to a 3rd party - often centralized and a single point of failure
1. Use an open market solution - decentralized, but can come with complex incentive alignment, the potential for competitive bots that increase the cost of execution, and difficulties ensuring reliability
1. Use the Chainlink Keeper Network```

**Review by AnushV — APPROVED** [review]
Looks good, lets get this out.

**Pull request merged**